src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1951,7 +1951,7 @@ namespace ReportSizingManager {
 					}
 					AutosizeDes *= StdRhoAir;
 				} else if (SizingType == WaterHeatingCoilUASizing) {
-					if ( DataCapacityUsedForSizing >= SmallLoad ) {
+					if( DataCapacityUsedForSizing >= SmallLoad && DataWaterFlowUsedForSizing > 0.0 && DataFlowUsedForSizing > 0.0 ) {
```

Comment:
Space between if and open parenthesis got lost, but otherwise these code changes look good.  Diffs in DElight files do not bother me.  I'm OK merging this.  Any hold ups from anyone else?